CMakeLists.txt

```diff
@@ -320,9 +331,9 @@ else() # Standalone build
 		PRIVATE ${AER_COMPILER_DEFINITIONS})
 	if(WIN32 AND NOT BLAS_LIB_PATH)
```

Comment:
Also wondering if we can simplify the `BLAS_LIB_PATH` logic now that we have a way to disable Conan. 
I'm concerned about the complexity of the build system in general, and having a way to skip over BLAS, and skip over everything else looks confusing, doesn't it?. 
What about removing the `BLAS_LIB_PATH` and if someone wants to build with their own BLAS, force them to go through the disable Conan path?. Certainly it will require them to have not just BLAS but every dependency installed.
Thoughts?

Reply:
Not sure, because this was requested in order to use the same `BLAS` library as the one `Numpy` uses. So probably not in the usual paths where CMake will look for. Not sure if this option is very used, though. 
But yeah, I agree, the build system is quite complex.

Reply:
Not sure, because this was requested in order to use the same `BLAS` library as the one `Numpy` uses. So probably not in the usual paths where CMake will look for. Not sure if this option is very used, though. 
But yeah, I agree, the build system is quite complex.